### ISSUE TITLE
Hamburger menu keyboard tabbing

### ISSUE THREAD

**Issue by Charcharbinkz** [cds-snc/digital-canada-ca#4474]
Noticing two things with keyboard tabbing as it relates to our hamburger menu.

1. Unable to tell it is keyboard focus. I think the border around the button might make it blend in. Wondering if it's possible to make it more obvious?

Selected:
<img width="726" alt="Screenshot 2023-01-09 at 10 27 34 AM" src="https://user-images.githubusercontent.com/42043272/211344735-eb6990cd-4c09-4267-876a-66b729602e05.png">

Not selected:
<img width="769" alt="Screenshot 2023-01-09 at 10 27 48 AM" src="https://user-images.githubusercontent.com/42043272/211344774-d3fd6b32-cb50-4459-a975-c7466ab3c213.png">

2. Looks like when the modal pops up, the options aren't selectable via tabbing. It continues to tab through the homepage underneath.
